modules/cdp-engine/src/lib.rs

```diff
@@ -112,8 +112,9 @@ type ChangeBalance = Change<Balance>;
 /// Liquidation strategy available
 #[derive(Encode, Decode, Clone, RuntimeDebug, PartialEq, Eq, TypeInfo)]
 pub enum LiquidationStrategy {
-	/// Liquidation CDP's collateral by create collateral auction
-	Auction,
+	/// Liquidation CDP's collateral by create collateral auction, u32 represents number of auctions
+	/// created
+	Auction(u32),
 	/// Liquidation CDP's collateral by swap with DEX
```

Comment:
I believe this type would need to be updated in acala.js, if this is reaasonable

Reply:
no need to update types with metadata v14